src/jobservice/logger/factory.go

```diff
@@ -81,5 +81,9 @@ func DBFactory(options ...OptionItem) (Interface, error) {
 		}
 	}
 
+	if len(key) == 0 {
```

Comment:
Is there any default value for `key` if user doesn't set it in `config.yaml`?

Reply:
this is a Job UUID, not config parameters. and this is the unique index for DB log data, must exists.

Reply:
thanks